acpi/src/bgrt.rs

```diff
@@ -15,7 +15,10 @@ pub struct Bgrt {
     image_offset_y: u32,
 }
 
-impl AcpiTable for Bgrt {
+// ### Safety: Implementation properly represents a valid BGRT.
```

Comment:
(this applies to both current PRs) how does this look rendered in `rustdoc`? If the whole line appears as the heading then it'd probably be better to put the details on a new line?